plugins/yoga/tests/plugin-graphql-yoga.test.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  InMemorySpanExporter,
+  TRACER_NAME,
+  setupOtel,
+} from "@graphql-debugger/opentelemetry";
+
+import { makeExecutableSchema } from "@graphql-tools/schema";
+import express from "express";
+import supertest from "supertest";
+
+import { createYoga } from "../src";
+
+const exporter = setupOtel({
+  inMemory: true,
+}) as unknown as InMemorySpanExporter;
+
+const typeDefs = /* GraphQL */ `
+  type Query {
+    hello: String!
+  }
+`;
+
+const resolvers = {
+  Query: {
+    hello: () => "Hello world!",
+  },
+};
+
+const schema = makeExecutableSchema({
+  typeDefs,
+  resolvers,
+});
+
+describe("plugin-graphql-yoga", () => {
+  afterEach(() => {
+    exporter.reset();
+  });
+
+  test("should wrap graphql yoga and assert spans are created on a query", async () => {
+    const yoga = createYoga({
+      schema,
+    });
+    const app = express();
+    app.use(express.json());
+    app.use("/graphql", yoga);
+
+    const query = /* GraphQL */ `
+      query {
+        hello
+      }
+    `;
+
+    await supertest(app)
```

Comment:
You can use `yoga.fetch` instead of setting up an express app, or if you still want to use `supertest`, you can pass `yoga` as a request listener to `supertest` directly.